pkg/proc/stack.go

```diff
@@ -377,7 +377,10 @@ func (it *stackIterator) appendInlineCalls(frames []Stackframe, frame Stackframe
 		if !okname || !okfileidx || !okline {
 			break
 		}
-		if fileidx-1 < 0 || fileidx-1 >= int64(len(frame.Current.Fn.cu.lineInfo.FileNames)) {
+		var bi *BinaryInfo
+		var e *dwarf.Entry
+		filepath, fileOk := bi.cuFilePath(frame.Current.Fn.cu, int(fileidx), e)
```

Comment:
It's pretty weird that it calls a method on a guaranteed nil object that is never used by the method. I think it should be a method of compileUnit. Instead of the logging inside cuFilePath it could return an error that the invocation in bininfo.go will print and here will just be ignored (`if err != nil { break }`).

Reply:
Agreed, the nil receiver is a bit weird. OK, I'll convert to compileUnit method and move the logging outside. I'll work on updating that.